docs/components/stars.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, minimum-scale=1.0, maximum-scale=1.0, user-scalable=no">
+  <title>Stars | Coco &amp; Eve Style Guides</title>
+
+  <link rel="stylesheet" href="/cocoandeve-styleguides/css/main.critical.css">
+  <link rel="stylesheet" href="/cocoandeve-styleguides/css/main.css">
+</head>
+
+<body class="mobile-wrapper">
+  <div class="container mt-4">
+    <h1>Review Stars</h1>
+    <span class="review-stars">
```

Comment:
all stars will have the same color inside the review-star component, so star-primary should be applied only once on the main element as review-stars--primary.
please also change the name of the file in review-stars.html